packages/widget/src/components/common/dropdown/dropdown.ts

```diff
@@ -36,6 +37,9 @@ export class Dropdown extends BaseComponent {
   @property({ type: Array })
   options: DropdownOption[] = [];
 
+  @property({ type: Object })
+  actionOption?: HTMLTemplateResult | null = null;
```

Comment:
does it makes sense for this to be optional if by default is initialized as null?

Reply:
let's keep the null only